docker-compose.yml

```diff
@@ -28,32 +33,6 @@ services:
     ports:
       - "8100:80"
 
-  #               _
-  # __      _____| |__
-  # \ \ /\ / / _ \ '_ \
-  #  \ V  V /  __/ |_) |
-  #   \_/\_/ \___|_.__/
-  #
-
-  web:
-    image: metacpan/metacpan-web:latest
-    build:
-      context: ./src/metacpan-web
-    volumes:
-      - type: volume
-        source: web_carton
-        target: /carton
-      - type: bind
-        source: ./src/metacpan-web
-        target: /metacpan-web
-        # read_only: true
-    ports:
-      - "5001:80"
-    networks:
-      - web-network
-    environment:
-      - PLACK_ENV=development
```

Comment:
We need this for debug output.

Reply:
metacpan-web's docker-compose.yml [uses the `develop` target](https://github.com/metacpan/metacpan-web/blob/master/docker-compose.yml#L13), which [sets PLACK_ENV=development](https://github.com/metacpan/metacpan-web/blob/master/Dockerfile#L66-L69).